src/main/java/org/springframework/data/elasticsearch/core/query/BaseQuery.java

```diff
@@ -26,6 +26,7 @@
 import java.util.Optional;
 import java.util.stream.Collectors;
 
+import org.elasticsearch.search.builder.SearchSourceBuilder;
```

Comment:
no Elasticsearch imports in this class, these imports reference deprecated Elasticsearch classes and cannot be used with the new client.

Reply:
if it is deprecated then why is it not tagged? Keep in mind this pr is against `v4.3.x`